server/routes/induction/common/workedBeforeController.ts

```diff
@@ -32,6 +32,21 @@ export default abstract class WorkedBeforeController extends InductionController
     )
     return res.render('pages/induction/workedBefore/index', { ...view.renderArgs })
   }
+
+  updatedInductionDtoWithHasWorkedBefore(inductionDto: InductionDto, workedBeforeForm: WorkedBeforeForm): InductionDto {
```

Comment:
Moved from the Update subclass so that the Update and Create subclasses can use it